examples/kube-stack-private/README.md

```diff
@@ -0,0 +1,103 @@
+
+#### Create the cluster using FPCO Terraform modules
+
+First, review the variables defined in `variables.tf`. You can change their values by adding corresponding values in `vars.env` and `terraform-tfvars.tpl` as per the 3 examples currently in `vars.env`. Be sure to review the current values as well. In particular it is likely necessary that you choose a new value for the CoreOS AMI by choosing an HVM AMI for your particular region from the list for the stable channel found [here](https://coreos.com/os/docs/latest/booting-on-ec2.html) or [here](https://goo.gl/wLhUyH"). The version currently there is likely out of date.
+
+Once you've set up your variables, run these `make` targets:
+
+```bash
+ᐅ make render-tfvars
+ᐅ make network
+ᐅ make plan
+ᐅ make apply
+```
+
+After you have successfully completed creating the cluster infrastructure, add the following commands to your ssh configuration file `~/.ssh/config`:
+```
+Host 10.10.*
+  ProxyCommand ssh -W %h:%p bastion
+  IdentityFile ~/.ssh/id_rsa
+  User core
+Host ip-10-10-*
+  ProxyCommand ssh -W %h:%p bastion
+  IdentityFile ~/.ssh/id_rsa
+  User core
+Host bastion
+  hostname <your bastion ip here>
+  User ubuntu
+  IdentityFile ~/.ssh/id_rsa
+  ControlMaster auto
+```
+Be sure to substitute the IP address for the bastion host. If you aren't sure what that is, run the command `terraform show` and look for `bastion_ip`. If you used a different ssh key than the standard one, be sure to edit the config to reflect the key you actually used.
+
+##### Install Kubernetes Using Kubespray
+
+You are now ready to use Kubespray to install Kubernetes in the cluster you just created. Follow the following steps:
+1. Use `pip` to install the following Python packages: `boto3`,`os`,`json`,`argparse`,`netaddr`
+2. If you don't already have it installed, use pip to install [`ansible`](https://github.com/ansible/ansible)
+3. `git clone` the [Kubespray repository](https://github.com/kubernetes-incubator/kubespray/) from the Kubernetes Incubator to your local machine and then go into the created directory
+4. We need to create a directory where we can define the specific parameters for our own cluster. In the `inventory` directory, copy the sample directory to a directory named after whatever you want to name your cluster, as follows:
+```bash
+cp -R inventory/sample/ inventory/mycluster
+```
+5. One set of parameters we need to create our how to allocate the different EC2 for different purposes in the Kubernetes cluster. The Terraform setup has added tags to each instance to define its role in the cluster. We will use a Python script to dynamically create the inventory list (to read more about this, see [here](https://github.com/kubernetes-incubator/kubespray/blob/master/docs/aws.md). Copy the `kubespray-aws-inventory.py` Python script into the directory you just created:
+```bash
+cp contrib/aws_inventory/kubespray-aws-inventory.py inventory/mycluster/
```

Comment:
It might be worth noting that the default script from upstream does not differentiate between nodes in different clusters, so that can be annoying/problematic (unless I am mistaken and the inventory script was updated?).

Reply:
Since we only have one cluster in this example, it might be confusing to bring this point up here?